src/App.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
+import {Route, Redirect} from 'react-router-dom';
 
 import './App.css';
 import Header from "./components/Navigation/Header/Header";
-import MainBody from "./components/MainBody/MainBody";
+import MainBody from "./containers/MainBody/MainBody";
+import Welcome from "./containers/Welcome/Welcome";
+import AllArticles from "./containers/AllArticles/AllArticles";
```

Comment:
Do you think using the word `All` here adds any value to the naming? Shouldn't `Articles` do justice?